lms/djangoapps/discussion/rest_api/serializers.py

```diff
@@ -155,6 +155,7 @@ class _ContentSerializer(serializers.Serializer):
     anonymous_to_peers = serializers.BooleanField(default=False)
     last_edit = serializers.SerializerMethodField(required=False)
     edit_reason_code = serializers.CharField(required=False, validators=[validate_edit_reason_code])
+    edit_by_label = serializers.SerializerMethodField(required=False)
```

Comment:
why required=False?

Reply:
Added so that data can be deserialized without needing this field